### ISSUE TITLE
Elemental - Machine registration download button does not work anymore

### ISSUE THREAD

**Issue by juadk** [rancher/dashboard#6710]
**Setup**
- Rancher version: 2.6.7
- Rancher-dashboard branch: elemental-dev

**Describe the bug**
Machine registration download button does not work anymore:
![image](https://user-images.githubusercontent.com/6025636/185888056-55e6376a-44c9-4f37-8331-ce9fc63039e1.png)

In the console:
`Failed to load resource: the server responded with a status of 500 ()`

I targeted the elemental-dev branch:
![image](https://user-images.githubusercontent.com/6025636/185888663-5cb02475-558f-4c35-ba04-ada0f8eae0b8.png)

If you don't want to do it with UI, It can also be configured during the Rancher installation with:
```
 --set "extraEnv[0].name=CATTLE_UI_DASHBOARD_INDEX" \
 --set "extraEnv[0].value=https://releases.rancher.com/dashboard/elemental-dev/index.html" \
 --set "extraEnv[1].name=CATTLE_UI_OFFLINE_PREFERRED" \
 --set "extraEnv[1].value=Remote" \
```
I reproduced it with both.

**Comment by ldevulder** [comment]
Yes I agree that these menus are not really useful IMO. For example the `MachineInventorySelectorTemplate` can be defined directly when the cluster is created (as @juadk said) but the `MachineInventorySelector` entries are then created automatically from the `Template`. So not really useful to create them manually, and it can be denagerous to edit them in fact.
